logic/generateLogic.gradle

```diff
@@ -84,16 +85,16 @@ def createGenerateLogicTaskForPackage(
             def identifier = /[a-zA-Z0-9]+/
             def newLine = /(?:\r\n?|\n)/
             def newLineAndIndent = /$newLine\s*/
-            def parameter = { paramNumber -> /,(?: |$newLineAndIndent)(?<paramName$paramNumber>$identifier): (?<typeName$paramNumber>[^:]+?)/}
+            def parameter = { paramNumber -> /,(?: |$newLineAndIndent)(?<vararg$paramNumber>vararg )?(?<paramName$paramNumber>$identifier): (?<typeName$paramNumber>[^:]+?)/}
```

Comment:
revert, not required if we don't use varag in logic

Reply:
Done